### PR TITLE
TabView v2 (#70)

### PR DESCRIPTION
**Pull request by stmoy**
* Move completed TabView v1 docs to archive
* Document the TabView v2 features

### CODE REVIEW

File: active/TabView/TabView_v2.md [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+[WUXC_VERSION_MUXONLY]
+[webhosthidden]
+enum TabViewWidthMode
+{
+    Equal = 0,
+    SizeToContent = 1,
+    Compact = 2,
+};
+
+[WUXC_VERSION_PREVIEW]
```
Comment (chingucoding) [inline active/TabView/TabView_v2.md]: Should this be preview or should it be [WUXC_VERSION_MUXONLY] ? I thought the goal is to finalize this API so we can mark it as public, having it be marked as preview here is a bit counterintuitive in my opinion.

File: active/TabView/TabView_v2.md [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+};
+
+[WUXC_VERSION_MUXONLY]
+[webhosthidden]
+[contentproperty("TabItems")]
+unsealed runtimeclass TabView : Windows.UI.Xaml.Controls.Control
+{
+
+    [WUXC_VERSION_PREVIEW]
+    [MUX_DEFAULT_VALUE("winrt::TabViewCloseButtonOverlayMode::Always")]
```
Comment (oldnewthing) [inline active/TabView/TabView_v2.md:116]: Shouldn't the default be `Auto`, not `Always`? Otherwise, what's the point of `Auto`?

Comment (chingucoding) [reply active/TabView/TabView_v2.md:116]: Yes, `Auto` should be the default

File: active/TabView/TabView_v2.md (hunk 1 above)
Comment (stmoy) [reply active/TabView/TabView_v2.md:100]: cc: @Mikehillberg - I think it should be WUXC_VERSION_MUXONLY?

Comment (stmoy) [reply active/TabView/TabView_v2.md:100]: cc: @Ranjeshj

Comment (ranjeshj) [reply active/TabView/TabView_v2.md:100]: Yes. Once the review is complete, we can switch the attribute from preview to WUXC_VERSION_MUXONLY. Thanks!

**Pull request merged**